README.md

```diff
@@ -103,9 +103,15 @@ It will output details for each request/response similar to the following snippe
 
 ### HTTP Endpoints
 
-If the model server is up and running, you can access the model server API via `http://localhost:8081/api/v1/*`.
+The Model Server supports two versions of the API (v1 and v2). If the model server is up and running, you can access the model server v1 API via `http://localhost:8081/api/v1/*`,
+and the v2 API via `http://localhost:8081/api/v2/*`
 
-The following table shows the current HTTP endpoints:
+#### HTTP Endpoints - V1
+
+<details>
```

Comment:
I don't have a strong preference. Just following @ndoschek 's suggestion. Although having two very similar tables in the same page would IMO be confusing, as you would always have to double-check you're looking at the correct version. Having a more detailed REST API Documentation would make sense (@ndoschek suggested using Javalin OpenAPI in #170), and then we could remove these tables entirely from the README (Or replace them with the generated pages, depending on how large they are; but I think referencing the pages makes more sense).

Reply:
Ok, thank you for clarifying that for me. Of course a generated API of some sort would be even better. I do not have too strong an opinion on this and since there is an open documentation task I am fine with how things are at the moment.